### PR TITLE
pipeline for rados, separate directories for librbd and rados

### PR DESCRIPTION
**Pull request by dthpulse**

### CODE REVIEW

File: pipeline/general.conf [hunk 1]
```diff
@@ -6,7 +6,7 @@ filter {
 }
 
 output {
-	#stdout { codec => rubydebug }
+	stdout { codec => rubydebug }
```
Comment (denisok) [inline pipeline/general.conf]: you forgot to comment this :)

File: pipeline/cbt_librbd.conf [hunk 2]
```diff
@@ -1,7 +1,7 @@
 input {
 	file
 	{
-	 	path => [ "${LOGSTASH_DIR}/**/results/**/**/json_output.*" ]
+	 	path => [ "${LOGSTASH_DIR}/librbd/**/results/**/**/json_output.*" ]
```
Comment (denisok) [inline pipeline/cbt_librbd.conf:4]: that librbd prefix is extra to cbd results, please add to the doc that... That someone need to create those dirs.

File: pipeline/cbt_rados_ses7.conf [hunk 3]
```diff
@@ -0,0 +1,34 @@
+input {
```
Comment (denisok) [inline pipeline/cbt_rados_ses7.conf]: could we remove ses7 ? it shouldn't be specific for some release.

**Review by denisok — CHANGES_REQUESTED** [review]

**Comment by dthpulse** [comment]
requested fixes implemented

**Review by denisok — APPROVED** [review]

**Pull request merged**